calcom/cal.com
# Fix all day events from Outlook allowing bookings

## What does this PR do?



This is a hot fix to not allow bookings for all day events. Since busy times coming from MS Graph were in UTC, converted to a dayjs object, back to a string, it would apply the UTC offset to the time which allowed some bookings through.

Fixes issue from email thread.

## Type of change




## How should this be tested?



- Connect an Outlook calendar
- Create an all day event on Outlook
- Go back to cal.com and on the availability page select the day that you made the all day event
- You should see no times for that day
